tests/migrations/test_references.py

```diff
@@ -0,0 +1,127 @@
+from django.db.backends.base import references
+from django.test import SimpleTestCase
+
+
+class TableTests(SimpleTestCase):
+    def setUp(self):
+        self.reference = references.Table('table', lambda table: table.upper())
```

Comment:
Are these reference objects created in setUp ever mutated by a test? If not, they could be created at the class-level rather than in setUp.